src/Jack/Property.purs

```diff
@@ -226,3 +230,21 @@ printSampleTree gen = do
   forest <- map (List.take 1) $ sampleTree 10 1 gen
   for_ forest $ \tree -> do
     log $ show tree
+
+jEq :: forall a. (Eq a, Show a) => a -> a -> Property
+jEq x y =
+  counterexample "=== Not equal ===" $
+  counterexample render (property (x == y))
+  where
+    render = show x <> " /= " <> show y
```

Comment:
Just one nit, no `where` clauses in this repo :)

so can you change it to something like:
```purescript
  let
    render =
      show x <> " /= " <> show y
  in
    counterexample "=== Not equal ===" $
    counterexample render (property (x == y))
```

cheers